index.html

```diff
@@ -61,12 +61,11 @@ <h3>Haiku</h3>
                 <h3>Public Domain Audiobook</h3>
                 <audio 
                     src= "https://ia601403.us.archive.org/31/items/treachery_in_outer_space_2010_librivox/treacheryouterspace_01_rockwell_128kb.mp3" controls>
+                    <a href="https://librivox.org/treachery-in-outer-space-by-carey-rockwell/"> Download Link</a>
                 </audio>
-                <a href="https://librivox.org/treachery-in-outer-space-by-carey-rockwell/"> Download Link</a>
-                <video 
-                    src= "MOO.mp4" controls width= "600" height= "500" preload= "auto"><a href= "https://www.youtube.com/watch?v=I4Q3YDezqcM&t=2s">Watch the Video</a>
-                </video>
```

Comment:
To embed a YouTube video, just add the iframe given by YouTube when you click the share button on the video